src/planscape/rscripts/forsys.R

```diff
@@ -10,9 +10,12 @@ library("glue")
 library("forsys")
 library("sf")
 library("dplyr")
+library("purrr")
```

Comment:
Do we need to update the getting started wiki, which contains a list of R libs to install?

Reply:
![bEvcrnYmh4fhRsX](https://github.com/OurPlanscape/Planscape/assets/125416076/536d1bb3-fac5-4db6-b863-25909b8f3d81)

Reply:
Yes, another PR is fine.